employees/tests/test_report_list_views.py

```diff
@@ -116,7 +116,7 @@ def test_author_report_list_view_should_display_monthly_hour_sum_in_hour_format(
 class ProjectReportListTests(TestCase):
     def setUp(self):
         super().setUp()
-        self.task_type = TaskActivityType(pk=1, name="Other", is_default=True)
+        self.task_type = TaskActivityType(pk=1, name="Other", is_active=True)
```

Comment:
Did you check if other properties here are still necessary? I remember that adding `pk=1` and `name="Other"` here was done here because of some workaround but I don't really remember why now. Please check if it's still needed